VMEncryption/main/Common.py

```diff
@@ -109,6 +109,7 @@ class CommonVariables:
     UpdateEncryptionSettings = 'UpdateEncryptionSettings'
     DisableEncryption = 'DisableEncryption'
     QueryEncryptionStatus = 'QueryEncryptionStatus'
+    Migrate = 'Migrate'
```

Comment:
With the introduction of a new "nonquery" operation type, it would be nice to include unit tests around this type to make sure we aren't breaking any configuration settings parsing logic written before the introduction of this new op